manifests/plugin/remote_execution/ssh/params.pp

```diff
@@ -0,0 +1,13 @@
+# Remote Execution SSH default parameters
+class foreman_proxy::plugin::remote_execution::ssh::params {
+  include ::foreman_proxy
```

Comment:
Kafo doesn't resolve `$::foreman_proxy` values below unless I do this - is there a better way?